com.playeveryware.eos/Documentation~/full_guide/01_starthere.md

```diff
@@ -0,0 +1,126 @@
+Custom policies can be defined here, and changed at any time. You don't need to commit to the final details at this time.
+The most permissive default policy is `GameClient /w UnlockAchievements`, which will give the Client Policy all of the permissions necessary to perform most actions.
+
+When the Client Policy is created, next create a client. Press `Add new client` to bring the Add new client dialogue.
+This will require a name and a Client Policy. Use the previously created Client Policy.
+`Trusted Server IP allow list` and `Epic Account Service Redirect URL` are not required to be filled at this time.
+
+# Creating an Application
+
+Navigate to the `Epic Account Services` page. The landing will show you all configured Applications, which is empty at first.
+See [Epic's documentation on Applications in the Developer Portal Introduction](https://dev.epicgames.com/docs/dev-portal/dev-portal-intro#application) for more information on Applications.
+An Application is collection of information about your game, which is later used to provide the Epic Game Store with essential information and permissions.
+Use `Create Application` to get started. Immediately when you press this button, an Application will be created for you.
+This will bring up a dialogue with three major steps; `Brand Settings`, `Permissions`, and `Linked Clients`.
+
+`Brand Settings` contains information about your game such as the Application's website.
+To utilize the EOS Plugin for Unity, you do not need to fill this page out at this time.
+
+`Permissions` will set the essential information that each user needs to consent to in order to use your application.
+This can be adjusted later. For full use of the EOS Plugin for Unity, consider marking `Online Presence` and `Friends` as Required.
+
+`Linked Clients` matches an Application to the Client that dictates its policies and further permissions.
+Assign the Client Policy created earlier to this.
+
+When you exit this dialogue, the created Application will appear in the Applications page.
+Navigating back to the `Product Settings` page, `Sandboxes` and `Deployments` will automatically be made for Dev, Stage, and Live.
+
+# Add the EOS Plugin for Unity to Your Game
+
+With a Client Policy, Client, and Application created, you're ready to start adding the EOS Plugin for Unity to your game.
+Follow [the Add Plugin documentation](/com.playeveryware.eos/Documentation~/add_plugin.md) to add the Plugin to the project.
+Once successfully included, follow [the guidance on Importing the Samples](com.playeveryware.eos/Documentation~/samples.md).
+In the EOS Plugin for Unity project, the provided code with the plugin will give you the ability to use EOS SDK's interfaces using convenient C# wrapper.
+Inside the Samples are a set of `Manager`s and `Service`s that wrap up the EOS SDK in easy-to-use functions.
+The Samples come with scenes that demonstrate the usage of the EOS Plugin for Unity's Manager classes for each Sample's domain.
```

Comment:
The Samples _are_ the scenes - this implies that the samples are distinct from the sample scenes.

Reply:
Mmm.... I'm not sure I agree. When a user has the Unity Package Manager to import samples, they get the scenes, but also they get our manager code and such. Someone can use the manager codes separate of the scenes. I think they are distinct.

Reply:
~The distinction that I would make is that our manager code and such is the _implementation of our plugin_, it is our efforts to wrap the EOS SDK comprehensively in a package that is consumable by anyone (generally speaking) with a Unity Project.~

~Our samples are not samples of the EOS SDK directly - but samples of how to use what we have implemented (that happens to wrap the EOS SDK).~

After writing this out I realize I was wrong. What is crossed out above is sort of what I think would be _ideal_, but that is not a reflection of reality right now. Your original text was spot on. Apologies for the confusion.